generators/app/tasks/createReactApp.js

```diff
@@ -2,7 +2,7 @@ const runCommand = require('./runCommand');
 
 module.exports.installCRA = function installCRA() {
   return runCommand({
-    command: ['npm', ['install', '--global', 'create-react-app@^3.0.0']],
+    command: ['npm', ['install', '--global', 'create-react-app']],
```

Comment:
SHould be hardcode to CRA 4 to avoid compatibility problems?

Reply:
Yeah, good catch